httpcore5/src/main/java/org/apache/hc/core5/http/HttpStatus.java

```diff
@@ -49,35 +47,35 @@
 public interface HttpStatus {
```

Comment:
We should NOT be using an interface to define constants, this should be a class. That's a class Effective Java item ;-) If this PR gets merged without that change. I can provide a separate PR.

Reply:
I love Joshua Bloch! @garydgregory Let's move this to a separate PR.